src/vnc.yml

```diff
@@ -0,0 +1,37 @@
+---
+- hosts: all
+  name: Install VNC
+  become: yes
+  become_method: sudo
+  roles:
+    - vnc
```

Comment:
I prefer [this sort of jazz](https://github.com/cisagov/kali-packer/blob/develop/src/assessor_tools.yml#L16-L25), to prevent unnecessary scope pollution.

Reply:
Scope pollution emissions reduced in https://github.com/cisagov/pca-gophish-composition-packer/pull/6/commits/1ec7a0683711da69333521f84b9f1f08b1ff4d34.  Thanks for the suggestion! 👍